### PR TITLE
Add Image Cutout Docs

### PR DESCRIPTION
**Pull request by noahhoffman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -821,6 +827,99 @@ There are a few limitations to the APIs you should be aware of ahead of time.
...
+The Image Cutout API is powered by Sensei, Adobe’s Artificial Intelligence Technology, and Photoshop. The API's can identify the main subject of an image and produce two types of outputs. You can create a greyscale [mask](https://helpx.adobe.com/photoshop/using/masking-layers.html) png file that you can composite onto the original image (or any other).  You can also create a cutout where the mask has already composited onto your original image so that everything except the main subject has been removed.
+
+| Original        | Mask           | Cutout  |
+| :-------------: |:-------------:| :-----:|
+| ![Alt text](assets/sensei_orig.jpg?raw=true "Original Image") | ![Alt text](assets/sensei_mask.png?raw=true "Mask") | ![Alt text](assets/sensei_cutout.png?raw=true "Original Image") |
+
+
+## General Workflow
+
+The typical workflow involves making an API POST call to the endpoint https://image.adobe.io/sensei for which the response will contain a link to check the status of the asynchronous job. Making a GET call to this link will return the status of the job and, eventually, the links to your generated output.
```
Comment (khound) [inline README.md]: @noahhoffman `https://image.adobe.io/sensei` ,clicking on this link returns 404. Should we mention,something like "API POST calls to the sensei hosted endpoints  https://image.adobe.io/sensei` ?

File: README.md [hunk 2]
```diff
@@ -821,6 +827,99 @@ There are a few limitations to the APIs you should be aware of ahead of time.
 
 The file Example.psd is included in this repository if you'd like to experiment with these example calls on your own.
 
+# ImageCutout
+
+The Image Cutout API is powered by Sensei, Adobe’s Artificial Intelligence Technology, and Photoshop. The API's can identify the main subject of an image and produce two types of outputs. You can create a greyscale [mask](https://helpx.adobe.com/photoshop/using/masking-layers.html) png file that you can composite onto the original image (or any other).  You can also create a cutout where the mask has already composited onto your original image so that everything except the main subject has been removed.
```
Comment (khound) [inline README.md]: @noahhoffman The `mask` hyperlink takes the user to the photoshop page. That page talks about both layer mask and vector mask. Do we need to mention here, what kind of mask the user needs to create?

**Comment by khound** [comment]
@noahhoffman Looks good. Just two small feedbacks.

**Comment by khound** [comment]
👍

File: README.md (hunk 1 above)
Comment (noahhoffman) [reply README.md]: @khound it sort of already says that, or am I missing something?

"The typical workflow involves making an API POST call to the endpoint https://image.adobe.io/sensei"....

Comment (khound) [reply README.md]: @noahhoffman My point is with `404` when a user clicks the link. If we mentioned somehow that, it's not an absolute link.That's why i put `the sensei hosted`,or we can just delete the link.

Comment (noahhoffman) [reply README.md]: @khound I see the possible source of confusion, I'm not sure that will help.  Let's see if it presents an issue for people

Comment (khound) [reply README.md]: Sure.

**Pull request merged**